src/compiler/utils/full_char_code_at.rs

```diff
@@ -0,0 +1,10 @@
+pub fn full_char_code_at(str: &str, i: usize) -> u8 {
```

Comment:
I think choose another name to make it easier to distinguish such as `word`

Reply:
Done